articles/from-enthusiasm-to-commercial-success.md

```diff
@@ -0,0 +1,74 @@
+**From Enthusiasm to Commercial Success**
+
+Through history, the goal of every human has changed. At the beginning, the sole
+goal was to survive. With evolution of mankind, big percent of people could work
+regularly without worrying about survival. Today, our work is not only the
+source of our income. We want to feel that we are contributing and that we
+matter. We want our ideas developed. To achieve this, there are different
+important things to understand.
+
+First, literally everything is achievable. But, there is also a price you need

```

Comment:
As I understand, this paragraph is kind of the introduction and sets the stage for a imaginary journey into "I want to achieve something great". Correct?

If this is the case, I would split the paragraph into two paragraphs:
a) the introduction
b) the first hint: You should not spend 100% of your day working .....

In the introduction, make clear that you are giving some hints/aspects/details/factors to succeed in achieving your goal. This is quite implicit now. Make it explicit.

And then I'd list the hints/aspects/details/factors of the following paragraphs in an unordered list

Reply:
The idea of this article is to give the very first hint. The hint was meant to be something like "you should take care about resources and effort you put into something". So, the hint is not only "You should not spend 100% of your day working...", it is also "Everything is achievable, for a certain price". But, I see that it needs adaptation because my message was not passed the way I wanted.

Reply:
I wrote another paragraph that lists all the aspects that will be mentioned. I am not sure whether I should add headings before every aspect, to make it even more explicit.

Reply:
I thinks its much clearer now with the introduction paragraph.